src/control/Control.Zoom.js

```diff
@@ -68,14 +69,16 @@ export var Zoom = Control.extend({
 	},
 
 	_zoomIn: function (e) {
+		var delta = any3d ? this._map.options.zoomDelta : 1;

```

Comment:
...but what if `zoomDelta` is **greater** than 1? There's an edge case of a map with only even (2,4,6,8,10,etc) zoom levels, and this would break that.

Maybe something like `delta = any3d ? zoomDelta : Math.ceil(zoomDelta)`??

Reply:
@IvanSanchez Yes, you are right if zoomDelta is bigger than 1 my fix has bug because delte will always be 1. I fixed it in new commit.
Also there will be no problems if zoomDelta will be bigger than 1 because map _limitZoom method just round it zoom buttons and double click will work.